docs/Differences_Ethereum_Overview.md

```diff
@@ -0,0 +1,62 @@
+---
+id: Differences_Overview
+sidebar_label: Overview
+title: Overview of Differences with Ethereum
+custom_edit_url: https://github.com/OffchainLabs/arbitrum/edit/master/docs/Differences_Ethereum_Overview.md
+---
+
+Arbitrum rollups aim to mantain compatibility with Ethereum. Smart contracts are compatible on the bytecode level, but there are certain aspects of the system that work differently to the EVM.
+
+Some of the opcodes have slightly different behaviours, as seen in [Solidity Support](Solidity_Support.md).  
+Concepts such as [Time in Arbitrum](Time_in_Arbitrum.md) and [Gas](ArbGas.md) play out differently in Layer 2.  
+Other differences are cool [Special Features](Special_Features.md) we squeezed in.
+
+## Ethereum Accounts
+
+### Nonces
+
+Every transaction submitted to Arbitrum will burn a nonce, except if the transaction is formatted incorrectly or does not have the expected nonce.
+
+### L1 to L2 Deposits
+
+Ether can be depositted using two methods: [retryable transactions](L1_L2_Messages.md) or L2 funded by L1 transactions. For end users these behave similarly, but have subtle differences.
+
+When depositing funds it is possible to send ether into a contract address without executing its fallback function - a scenario similar to when contracts _self destruct_, sending funds to a contract address.
+
+## JSON RPC API
+
+The API for Arbitrum aims to be a superset of the [eth spec](https://eth.wiki/json-rpc/API). When interacting with it you can expect all the usual fields, as well as some extra ones used to surface information unique to Arbitrum Rollups.
+
+### Transaction Receipts
+
+Transaction receipts contain the following extra fields
+
+#### L1 Block Number
+
+The Layer 1 block number for the transaction, as specified in [Time in Arbitrum](Time_In_Arbitrum.md).
+
+#### Fee Stats
+
```

Comment:
Maybe include table here too explaining each field? I think this would be quite nice, as quick shorthand way to understand the gas/fee architecture

Reply:
I think we should just have a whole page on the fee model, any shorthand here might just be lacking and more confusing imo